repo2docker/buildpacks/conda/__init__.py

```diff
@@ -279,7 +279,7 @@ def get_env_scripts(self):
                     "${NB_USER}",
                     r"""
                 TIMEFORMAT='time: %3R' \
-                bash -c 'time /tmp/bin/micromamba install -p {0} -f "{1}" && \
+                bash -c 'time /tmp/bin/micromamba env update -p {0} -f "{1}" && \
```

Comment:
I think we only have the `update` command (not `env update`).

Reply:
Yeah, I tried both, both of them fail locally for me.
The only thing that works is using `install` along with specifying `- python` in the `environment.yml` file

Reply:
I have still pushed the changes to see if it works with the CI, though it fails locally for me

Reply:
I am pushing a build with my changes for libsolv. Let's see if that will fix the issues here!